### DISCUSSION TITLE
Prism launcher

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [headlesshq/headlessmc#128] [Q&A]
How do I launch prism launcher instances with it?

**Comment by pizzav-xyz** [comment]
@3arthqu4ke

**Comment by 3arthqu4ke** [comment]
Idk, can the official Minecraft launcher launch prism instances?

**Reply by pizzav-xyz** [reply]
![image](https://github.com/3arthqu4ke/headlessmc/assets/103120356/520cd70e-2384-4198-b179-c3fcb9dae6a7)

**Reply by 3arthqu4ke** [reply]
then headlessmc cant either

**Reply by 3arthqu4ke** [reply]
Actually you could try to point `hmc.gamedir` to the directory the instance is stored in.
Not sure what will happen, but prism at least stores options and mods in those instance folders.
